RecoLocalMuon/RPCRecHit/python/rpcRecHits_cfi.py

```diff
@@ -15,3 +16,7 @@
 #disabling DIGI2RAW,RAW2DIGI chain for Phase2
 from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
 phase2_muon.toModify(rpcRecHits, rpcDigiLabel = cms.InputTag('simMuonRPCDigis'))
+from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
+(premix_stage2 & phase2_muon).toModify(rpcRecHits,
```

Comment:
educate me please what ```(premix_stage2 & phase2_muon) ``` syntax means

Reply:
both `premix_stage2` and `phase2_muon` eras must be chosen for the given modification to be applied.

Reply:
Let me also note that here (and in other places) the `phase2_muon` (/`phase2_common`) will be removed once we move the run2 premixing to store DIGIs instead of RAW.